src/envoy/mixer/config.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2017 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "precompiled/precompiled.h"
+
+#include "envoy/json/json_object.h"
+#include "include/attribute.h"
+
+namespace Http {
+namespace Mixer {
+
+// A config for mixer filter
+struct MixerConfig {
+  // the mixer server address
+  std::string mixer_server;
+
+  // These static attributes will be send to mixer in both
+  // Check and Report.
+  std::map<std::string, std::string> mixer_attributes;
```

Comment:
mixer_attributes will be send to mixer server
forward_attributes will be forwarded to upstream so that next hop proxy can send them to mixer server.  
Above attributes are for Check and Report.
Quota is a separate call.  Here we use some standalone attributes to trigger the calls.